azurerm_cdn_endpoint/main.tf

```diff
@@ -23,6 +23,29 @@ resource "azurerm_cdn_endpoint" "cdn_endpoint" {
     host_name = var.origin_host_name
   }
 
+  global_delivery_rule {
+    cache_expiration_action {
```

Comment:
You just make this block required while it is not in the terraform docs.
Maker this block optional creating a null default for the variable and using the for_each like this:
https://github.com/pagopa/io-infrastructure-modules-new/blob/master/azurerm_app_service/main.tf#L99

Reply:
Ok, it's correct. I changed the code as suggested. 
Actually I noticed this but I did not find the way to make the block optional and... at the end I was persuaded to make that block mandatory to require administrator to define a clear global max-age value (the default should be 7 days but it's not so clear).